### PR TITLE
Replace plus button in community by adhoc btn and add buton at the bottom

### LINKED ISSUE #5677: Remove + sign in Community and show create ad-hoc group button instead
Remove the + button at the top of communities and show the ad-hoc group button (same one that's in the personal chat section.

Also add a new "button" text at the button to add channels or categories

https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=2674%3A367792

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #5677

image

image

image

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/ChatColumnView.qml [hunk 1]
```diff
@@ -169,7 +169,7 @@ Item {
     CreateChatView {
         rootStore: root.rootStore
         emojiPopup: root.emojiPopup
-        visible: mainModule.activeSection.sectionType === Constants.appSection.chat && root.rootStore.openCreateChat
```
Comment (alaibe) [inline ui/app/AppLayouts/Chat/views/ChatColumnView.qml:172]: could that not open the create chat view when not in chat section?

File: ui/app/AppLayouts/Chat/views/CommunityColumnView.qml [hunk 2]
```diff
@@ -402,6 +386,51 @@ Item {
         }
     }
 
+    Loader {
+        id: createChatOrCommunity
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Style.current.padding
+        active: communityData.amISectionAdmin && communityData.canManageUsers
```
Comment (alaibe) [inline ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:394]: the `enabled` property of the menu item seems to indicate that u just need `amISectionAdmin`.
If I have `canManageUsers` only then there will be not menu item?
Seems odd?

**Review by saledjenic — APPROVED** [review]

**Review by iurimatias — DISMISSED** [review]
https://user-images.githubusercontent.com/176720/168130264-25ecf1b0-2dd9-4f44-b1b0-2057fbbfb945.mov

**Comment by jrainville** [comment]
@iurimatias I fixed it. It required a bigger change. Because the CreateChatView used to be part of all chat sections, but only works for the personal section, so I had to move it up to the AppMain. Then I handle opening it using a Global signal.
Re-requesting reviews because of that.

**Review by alaibe — APPROVED** [review]

**Comment by elina2015** [comment]
@jrainville hey, I found the next issues in this build (tested on macOS Monterey 12.3 Beta (21E5227a), and macOS Big Sur v.11.6.6):
1. I cannot accept the private contact request:
https://user-images.githubusercontent.com/14942081/168677078-0d42cf23-6aba-4959-aabb-4a17603ab2ed.mov
2. The buttons are shown very close:
image
3. The form is shown very close to the window's edge: 
image
4. After hover over the buttons, the pointer doesn't change it's view and for user it doesn't look like a clickable button:
image

Could you kindly check it?

**Review by saledjenic — APPROVED** [review]

**Comment by jrainville** [comment]
@elina2015 the issues you found should be opened in another issue, since they are not related to this PR as far as I can tell.

**Comment by jrainville** [comment]
@elina2015 the problem number two about the menu is fixed

**Review by elina2015 — APPROVED** [review]

**Pull request merged**